php_fpm/check.py

```diff
@@ -117,7 +121,9 @@ def _process_ping(self, ping_url, ping_reply, auth, tags, pool_name, http_host,
             # TODO: adding the 'full' parameter gets you per-process detailed
             # informations, which could be nice to parse and output as metrics
             resp = requests.get(ping_url, auth=auth, timeout=timeout,
-                                headers=headers(self.agentConfig, http_host=http_host))
+                                headers=headers(self.agentConfig,
```

Comment:
I think we have a typo here, I was expecting:
```
 resp = requests.get(ping_url, auth=auth, timeout=timeout,
                                   headers=headers(self.agentConfig, http_host=http_host)),
                                   verify=not disable_ssl_validation))
```
Just like we have in `_process_status()`, I believe verify is a parameter for the `requests` `get()` call.

Reply:
🤦‍♂️ good catch!